.eslintrc.js

```diff
@@ -4,6 +4,9 @@ module.exports = {
         "node": true,
         "es6": true,
     },
+    "parserOptions": {
+      "ecmaVersion": 8,
```

Comment:
you need to also update the Dockerfile node version :)

Reply:
done